src/share/poudriere/common.sh

```diff
@@ -8449,6 +8449,17 @@ svn_git_checkout_method() {
 				;;
 			esac
 			;;
+		pkgbase*)
+			case "${SOURCES_URL}" in
+			http://*) ;;
```

Comment:
does `http+pkg://` not work?

Reply:
probably does, let's first fix this feature and we can later expand it.
Right now it's not possible to give a full pkg repo description (with keys etc ...) so we might change again the format later.